datastore/integration-test/response/items_within_area_single_platform.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
Can we use the same result file as for `items_get_single_platform.json` by tweaking the bbox?